content/doc/book/installing/index.adoc

```diff
@@ -224,87 +491,59 @@ capabilities, rather than relying on the `SunOS` check, the above caveats for
 ZFS integration with Jenkins should be considered.
 
 
-=== macOS
-
-To install from the website, using a package:
-
-* link:http://mirrors.jenkins.io/osx/latest[Download the latest package]
-* Open the package and follow the instructions
-
-Jenkins can also be installed using `brew`:
-
-* Install the latest release version
-[source,bash]
-----
-brew install jenkins
-----
-
-* Install the LTS version
-[source,bash]
-----
-brew install jenkins-lts
-----
-
-
-=== Windows
-
-To install from the website, using the installer:
-
-* link:http://mirrors.jenkins.io/windows/latest[Download the latest package]
-* Open the package and follow the instructions
-
-
-=== Docker
-You must have link:http://docker.io[Docker] properly installed on your machine.
-See the link:https://www.docker.io/gettingstarted/#h_installation[Docker installation guide] for details.
-
-
-First, pull the official link:https://hub.docker.com/r/jenkins/jenkins[jenkins] image from Docker repository.
-
-[source,bash]
-----
-docker pull jenkins/jenkins
-----
-
-Next, run a container using this image and map data directory from the container to the host; e.g in the example below `/var/jenkins_home` from the container is mapped to `jenkins/` directory from the current path on the host. Jenkins `8080` port is also exposed to the host as `49001`.
-
-[source,bash]
-----
-docker run -d -p 49001:8080 -v $PWD/jenkins:/var/jenkins_home -t jenkins/jenkins
-----
-
-=== Other
-See link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
-
-
 [[setupwizard]]
-== Post-install (Setup Wizard)
+== Post-installation setup wizard
 
-=== Create Admin User and Password for Jenkins
+After downloading, installing and running Jenkins using one of the procedures
+above, the post-installation setup wizard begins:
 
-Jenkins is initially configured to be secure on first launch.
-Jenkins can no longer be accessed without a username and
-password and open ports are limited. During the initial run of
-Jenkins a security token is generated and printed in the console
-log:
-
-----
-*************************************************************
-
-Jenkins initial setup is required. A security token is required to proceed.
-Please use the following security token to proceed to installation:
-
-41d2b60b0e4cb5bf2025d33b21cb
-
-*************************************************************
-----
-
-The install instructions for each of the platforms above includes the default location for when you can find this log output.
-This token must be entered in the "Setup Wizard" the first time you open the Jenkins UI.
-This token will also serve as the default password for the user 'admin' if you skip the user-creation step in the Setup Wizard.
-
-=== Initial Plugin Installation
+. When the *Unlock Jenkins* page appears:
```

Comment:
Can you structure this to not have text split across images in one list item.

Reply:
Done - yeah I agree - it's ugly and I've reverted this back to the format I used in the tutorial instructions.